src/components/ContractName.tsx

```diff
@@ -41,21 +38,13 @@ function ContractNameSuspended({
   clearType,
   network,
 }: ContractNameProps) {
-  const { emailDerivativeContracts = [], ERC721derivativeContracts = [] } =
-    useSnapshot(SealCredStore)
   // We will always get a string
-  let contractName = getContractName(address, network) as string
+  let contractName = getContractName(address, network, truncate) as string
```

Comment:
Why do we need to convert `contractName as string` here? As I can see, the call stack doesn't return `undefined`, so we don't need this?